src/server/ua_services_session.c

```diff
@@ -415,16 +415,26 @@ Service_CreateSession(UA_Server *server, UA_SecureChannel *channel,
 
 static UA_StatusCode
 checkSignature(const UA_Server *server, const UA_SecurityPolicy *securityPolicy,
-               void *channelContext, const UA_ByteString *serverNonce,
-               const UA_SignatureData *signature) {
+               void *channelContext, const UA_ByteString *serverNonce,const UA_UserTokenPolicy *userTokenPolicy,
+               const UA_ExtensionObject *userToken, const UA_SignatureData *signature) {
     /* Check for zero signature length */
     if(signature->signature.length == 0)
         return UA_STATUSCODE_BADAPPLICATIONSIGNATUREINVALID;
 
     if(!securityPolicy)
         return UA_STATUSCODE_BADINTERNALERROR;
 
-    const UA_ByteString *localCertificate = &securityPolicy->localCertificate;
+    const UA_ByteString *localCertificate;
```

Comment:
It is easier to read if we don't break up declaration and assignment of the variable.